src/desktop/apps/fairs/helpers/view_helpers.coffee

```diff
@@ -52,13 +53,21 @@ module.exports =
     fair.is_published and fair.profile?.is_published and fair.profile?.icon?.url
 
   isNotOver: (fair) ->
-    Date.parse(fair.end_at) > new Date
+    endOfFair = moment.utc(fair.end_at)
+    offset = endOfFair.utcOffset() 
+    endOfFair = endOfFair.endOf("day")
+    now = moment().utcOffset(offset)
+    now.diff(endOfFair) < 0
```

Comment:
I wonder if we can DRY ourselves with:
```coffeescript
isNotOver: (fair) -> not @isOver(fair)
```

Not sure how we want to handle the equal case, but it may not be critical.

Reply:
@sepans and I were wondering why it wasn't originally written that way, but we decided to just leave it the way it was.

Reply:
Yes our guess was that may be the `equal` case should neither be *over* nor *notOver* but don't know if it makes any difference.